src/components/reusable/textArea/textArea.ts

```diff
@@ -56,7 +56,9 @@ export class TextArea extends FormMixin(LitElement) {
     return html`
       <div class="text-area" ?disabled=${this.disabled}>
         <label class="label-text" for=${this.name}>
-          ${this.required ? html`<span class="required">*</span>` : null}
+          ${this.required
+            ? html`<span class="required" aria-label="Required">*</span>`
```

Comment:
No hard-coded strings, must be customizable for localization purposes.

Reply:
Addressed.